bcr_tests/.bazelrc

```diff
@@ -0,0 +1 @@
+build --experimental_enable_bzlmod
```

Comment:
Is there a reason for `bcr_tests` be a separate top level directory in the root rather than live under the general `tests` directory?

Reply:
Moved it, no particular reason - it just needs to be a separate directory.